app/views/posts/index.html.erb

```diff
@@ -1 +1,36 @@
-<h1>Here is a list of posts</h1>
\ No newline at end of file
+<div class="detailed-post">
+
+        <div class="detail-container">
+                <img src=<%=@user.photo %>>
+                <div class="detail-user">
+                    <h1><%= @user.name %></h1>
+                    <h3>Number of posts: <%= @posts.count %>
+                    </h3>
+                </div>
+        </div>
+
+    <div class="recent-posts">
+        <% @posts.each do |post| %>
+        <div class="one-comment">
+                <h2><%= post.title %>
```

Comment:
- [ ] As suggested by the previous reviewer, your design must match the wireframe provided. The image of the user at this moment is extremely large, and as a result distorted the page layout in a way. Kindly look at the wireframe below and make your layout to look like it. Styling is not required at this moment though, but the wireframe layout must be maintained.

### Your Design
![user-img1](https://github.com/HERMON-1995/Blog_app_project/assets/26597432/676de295-6518-4069-84a1-944c9dd07e9f)

![user-img2](https://github.com/HERMON-1995/Blog_app_project/assets/26597432/a008904c-911a-4cf6-b88a-2741d7defad2)

### Wireframe
![wireframe](https://github.com/HERMON-1995/Blog_app_project/assets/26597432/e0cd302f-b8f6-48bf-a156-0de412055898)

Reply:
Ohh sorry about that, last time I only implemented the image section on the home page. now users post image is also fixed, Thank you!